runtime/runtime/src/config.rs

```diff
@@ -189,6 +205,48 @@ pub fn exec_fee(
         DeleteAccount(_) => cfg.delete_account_cost.exec_fee(),
     }
 }
+
+pub fn prepaid_exec_fee(
+    config: &RuntimeFeesConfig,
+    action: &Action,
+    receiver_id: &AccountId,
+    current_protocol_version: ProtocolVersion,
+) -> Gas {
+    let exec_gas = exec_fee(config, action, receiver_id, current_protocol_version);
+
+    let extra_prepaid_gas = match action {
+        Action::DeleteAccount(DeleteAccountAction { beneficiary_id }) => checked_feature!(
```

Comment:
On the first glance, it seems like this could be `if checked_feature!()` as well?

Reply:
Yes, that's right